geo/examples/concavehull-usage.rs

```diff
@@ -66,8 +66,8 @@ fn main() -> std::io::Result<()> {
         .collect();
     let moved_v = move_points_in_viewbox(width as f64, height as f64, v.clone());
     let multipoint = MultiPoint::from(moved_v);
-    let concave = multipoint.concave_hull(2.0);
-    let convex = multipoint.convex_hull();
+    let concave = multipoint.concave_hull(2.0).unwrap(); // TODO: use ? here?
+    let convex = multipoint.convex_hull().unwrap(); // TODO: use ? here?
```

Comment:
TODO

Reply:
As with tests, I'm personally pretty OK with playing fast-and-loose in example applications, especially if it's more concise.